packages/app/src/app/components/Create/CreateBox.tsx

```diff
@@ -51,7 +51,7 @@ type CreateBoxProps = ModalContentProps & {
 export const CreateBox: React.FC<CreateBoxProps> = ({
   collectionId,
   type = 'devbox',
-  hasSecondStep = false,
+  hasSecondStep = true,
```

Comment:
I believe we can remove this prop, right?

Reply:
I'd keep it for possibly implementing a flow where the click on the template forks directly